sql/2024/fonts/performance/fonts_service.sql

```diff
@@ -0,0 +1,45 @@
+-- Section: Performance
+-- Question: Which services are popular?
+
+-- INCLUDE ../common.sql
+
+WITH
+pages AS (
+  SELECT
+    client,
+    COUNT(DISTINCT page) AS total
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-07-01'
+  GROUP BY
+    client
+),
+services AS (
+  SELECT
+    client,
+    SERVICE(url) AS service,
+    COUNT(DISTINCT page) AS count
+  FROM
+    `httparchive.all.requests`
+  WHERE
+    date = '2024-07-01' AND
+    type = 'font'
+  GROUP BY
+    client,
+    service
+)
+
+SELECT
+  client,
+  service,
```

Comment:
The 2022 query also included data on whether a page used a combination of self-hosted or hosted fonts. Can we include that again? It can paint a very different picture...

Reply:
I have updated the query to show the actual combinations. Please take a look.